tests/test_PsfFlux.py

```diff
@@ -32,6 +32,22 @@
                                   SingleFramePluginTransformSetupHelper)
 
 
+def compute_chi2(exposure, centroid, instFlux, maskPlane=None):
+    """Return the chi2 for the exposure PSF at the given position."""
+    psfImage = exposure.getPsf().computeImage(centroid)
+    scaledPsf = psfImage.array*instFlux
+    # Get a sub-image the same size as the returned PSF image.
+    sub = exposure.Factory(exposure, psfImage.getBBox(), lsst.afw.image.LOCAL)
+    if maskPlane is not None:
+        masked = np.logical_not(sub.mask.array & sub.mask.getPlaneBitMask(maskPlane))
```

Comment:
Shouldn't this be called `unmasked` or `nonmasked`? Or, per the [style guide](https://developer.lsst.io/cpp/style.html#negated-boolean-variable-names-must-be-avoided), don't include the `not` in the variable definition?

Reply:
I've renamed it `unmasked`: I think in this case that problem described in the style guide shouldn't be an issue.